PyInstaller/depend/imphook.py

```diff
@@ -574,8 +590,11 @@ def __init__(self):
         self._datas = {}
 
     def add(self, modname, binaries, datas):
-        self._binaries[modname] = binaries or []
-        self._datas[modname] = datas or []
+
+        self._binaries.setdefault(modname, [])
+        self._binaries[modname].extend(binaries or [])
```

Comment:
Could this be written as `self._binaries.setdefault(modname, binaries or [])`? That seems a bit clearer to me.

Reply:
@bjones1 this will cause type exceptions - binaries is a `set` when passed in.

Reply:
OK, makes sense.